OctoAwesome/OctoAwesome/FunctionalBlock.cs

```diff
@@ -18,11 +31,5 @@ public void Interact(GameTime gameTime, Entity entity)
             OnInteract(gameTime, entity);
         }
 
```

Comment:
Interact does the same as ComponentContainer<IFunctionalBlockComponent>.Interact which is hidden by this method(as the method in the base class is not virtual).
-> Delete Interact method from FunctionalBlock

Reply:
FunctionalBlock and Entity will be merged pretty soon in the future, so this will resolve itself